stdlib/universe/moving_average_test.go

```diff
@@ -840,8 +795,7 @@ func TestMovingAverage_Process(t *testing.T) {
 		{
 			name: "empty table",
 			spec: &universe.MovingAverageProcedureSpec{
-				Columns: []string{execute.DefaultValueColLabel},
-				N:       2,
+				N: 2,
 			},
```

Comment:
Is there an error if `N` is less than 2, including 1, 0, or negative?  Would be good to verify that a reasonable error is produced.